src/browser/webgl.rs

```diff
@@ -0,0 +1,332 @@
+use super::*;
+
+static VERTEX: &str = indoc! {"
```

Comment:
These can be in separate files that we include with `include_str!`, probably `src/browser/vertex.glsl` and `src/browser/fragment.glsl`. We can avoid using `indoc!` and also get syntax highlighting when editing them.